modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -14,90 +14,167 @@ Object Store provides these APIs:
 * https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/object-store-v2/[Object Store v2 API]
```

Comment:
You probably already know but the banner isn't working on the beta site.

Reply:
🤔